python/sparkdl/graph/spark_utils.py

```diff
@@ -0,0 +1,108 @@
+#
```

Comment:
we should add that in a comment here then. Maybe rename the module to `tensorframes_udf` if we really want this to be in tensorframes eventually. `spark_utils` sounds like it's got a bunch of helper functions around spark.